Makefile

```diff
@@ -318,9 +318,6 @@ bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metada
 	cd config/manager && GOFLAGS="-mod=mod" $(KUSTOMIZE) edit set image controller=$(IMG)
 	GOFLAGS="-mod=mod" $(KUSTOMIZE) build config/manifests | GOFLAGS="-mod=mod" $(OPERATOR_SDK) generate bundle -q --extra-service-accounts "velero" --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
 	@make nullables
-	# Copy updated bundle.Dockerfile to CI's Dockerfile.bundle
-	# TODO: update CI to use generated one
-	cp bundle.Dockerfile build/Dockerfile.bundle
```

Comment:
@shubham-pampattiwar Can you confirm this is not needed? (we could also remove the doc file I modified)

Reply:
Usually most of that Makefile is generated by operator-sdk, did you experiment with creating new one or upgrading to the newer version of operator-sdk and re-generating Makefile?

Reply:
yes, we modified it https://github.com/operator-framework/operator-sdk/blob/e001847a67f16d7173ae4fee7cff57cd4405edbb/internal/plugins/manifests/v2/init.go#L223-L238